Pod/Classes/UI/CardTextField.swift

```diff
@@ -244,13 +267,16 @@ public class CardTextField: UITextField, NumberInputTextFieldDelegate {
         accessoryButtonLeadingConstraint?.constant = accessoryButtonLeadingInset
         accessoryButtonTrailingConstraint?.constant = accessoryButtonTrailingInset
         
+        // Reset gesture recognizers
+        [firstObjectInNib, cardInfoView].forEach({$0?.gestureRecognizers = []})
+        
         let leftSwipeGestureRecognizer = UISwipeGestureRecognizer(target: self, action: #selector(moveNumberFieldLeftAnimated))
-        leftSwipeGestureRecognizer.direction = .Left
+        leftSwipeGestureRecognizer.direction = isRightToLeftLanguage ? .Right : .Left
```

Comment:
Just an idea. Would something like `swipeToDetailInputTextFieldGestureRecognizer` be a better name? Same as for the `rightSwipeGestureRecognizer` vs `swipeToNumberInputTextFieldGestureRecognizer`